### DISCUSSION TITLE
"url" value in data section from Get Issued Document

### DISCUSSION THREAD

**Discussion by zpj61** [fattureincloud/api#185] [Q&A] [answered]
This value has a pdf file name, but how can I use this file name? And what is this pdf? Invoice's PDF?

**Comment by emajo** [comment, marked as answer]
Hi @zpj61 you can use `url` to download the invoice's PDF, keep in mind that is a temporary url so every time you need to download it you need to make the Get Issued Document call to retrieve a valid url.

Let us know if you have some other doubt,
Emanuele

**Reply by valmoz** [reply]
Hi @zpj61, do you have further questions regarding this topic?

Thanks
Mauro
